kubeflow/apps/pipelines/pull-upstream.sh

```diff
@@ -17,10 +17,10 @@
 set -ex
 
 # TODO(Bobgy): use KFP 1.6.1 when https://github.com/kubeflow/pipelines/pull/5750 is released.
-# export KUBEFLOW_PIPELINES_VERSION=1.5.0
-# export KUBEFLOW_PIPELINES_REPO=https://github.com/kubeflow/pipelines.git
-export KUBEFLOW_PIPELINES_VERSION=config-default-pipeline-root
-export KUBEFLOW_PIPELINES_REPO=https://github.com/Bobgy/pipelines.git
+export KUBEFLOW_PIPELINES_VERSION=1.7.0
+export KUBEFLOW_PIPELINES_REPO=https://github.com/kubeflow/pipelines.git
+# export KUBEFLOW_PIPELINES_VERSION=config-default-pipeline-root
+# export KUBEFLOW_PIPELINES_REPO=https://github.com/Bobgy/pipelines.git
```

Comment:
remove the reference to my fork repo?

Reply:
Sounds good, will make this change in next PR.